src/main/java/WayofTime/bloodmagic/item/ItemBoundTool.java

```diff
@@ -250,4 +250,24 @@ protected static void dropStacks(Multiset<ItemStackWrapper> drops, World world,
                 world.spawnEntity(new EntityItem(world, posToDrop.getX(), posToDrop.getY(), posToDrop.getZ(), stack.toStack(count)));
         }
     }
+
+    protected void sharedHarvest(ItemStack stack, World world, EntityPlayer player, BlockPos blockPos, BlockStack blockStack, HashMultiset drops){
+        boolean silkTouch = EnchantmentHelper.getEnchantmentLevel(Enchantments.SILK_TOUCH, stack) > 0;
```

Comment:
Silk Touch and Fortune levels should be passed in as arguments. The levels won't be changed between each block check.